docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -0,0 +1,230 @@
+[[url-drilldown]]
+=== URL Drilldown
+
+The URL drilldown allows you to navigate from a dashboard to an internal or external URL.
+The destination URL can be dynamic, depending on the dashboard context or user’s interaction with a visualization.
+
+For example, you might have a dashboard that shows data from a Github repository.
+You can create a drilldown that navigates from this dashboard to Github.
+Refer to the <<try-it, Try it section>> for instructions on how to create this drilldown.
```

Comment:
Remove "Refer to the try it section ..." because that section immediately follows this section